Found 2339 images belonging to 10 classes.
Found 50 images belonging to 10 classes.
Found 50 images belonging to 10 classes.
Epoch 1/11
2023-05-24 15:38:39.114410: I tensorflow/core/common_runtime/executor.cc:1210] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype int32
	 [[{{node Placeholder/_0}}]]
74/74 [==============================] - ETA: 0s - loss: 2.0954 - accuracy: 0.2240
2023-05-24 15:39:43.001679: I tensorflow/core/common_runtime/executor.cc:1210] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype int32
	 [[{{node Placeholder/_0}}]]
74/74 [==============================] - 64s 864ms/step - loss: 2.0954 - accuracy: 0.2240 - val_loss: 1.5930 - val_accuracy: 0.3400
Epoch 2/11
74/74 [==============================] - 66s 890ms/step - loss: 1.5847 - accuracy: 0.4198 - val_loss: 1.5522 - val_accuracy: 0.4200
Epoch 3/11
74/74 [==============================] - 67s 901ms/step - loss: 1.4920 - accuracy: 0.4446 - val_loss: 1.3389 - val_accuracy: 0.5200
Epoch 4/11
74/74 [==============================] - 68s 917ms/step - loss: 1.3395 - accuracy: 0.5024 - val_loss: 1.4083 - val_accuracy: 0.5200
Epoch 5/11
74/74 [==============================] - 69s 929ms/step - loss: 1.2837 - accuracy: 0.5515 - val_loss: 1.3015 - val_accuracy: 0.5800
Epoch 6/11
74/74 [==============================] - 81s 1s/step - loss: 1.1548 - accuracy: 0.5879 - val_loss: 1.3340 - val_accuracy: 0.5200
Epoch 7/11
74/74 [==============================] - 88s 1s/step - loss: 1.0782 - accuracy: 0.6229 - val_loss: 1.1581 - val_accuracy: 0.5800
Epoch 8/11
74/74 [==============================] - 80s 1s/step - loss: 0.9927 - accuracy: 0.6430 - val_loss: 1.1393 - val_accuracy: 0.6200
Epoch 9/11
74/74 [==============================] - 70s 946ms/step - loss: 0.9294 - accuracy: 0.6665 - val_loss: 1.4781 - val_accuracy: 0.4600
Epoch 10/11
74/74 [==============================] - 72s 964ms/step - loss: 0.9296 - accuracy: 0.6729 - val_loss: 1.3654 - val_accuracy: 0.6200
Epoch 11/11
74/74 [==============================] - 70s 942ms/step - loss: 0.8395 - accuracy: 0.7046 - val_loss: 1.0493 - val_accuracy: 0.6600

2023-05-24 15:51:54.815020: I tensorflow/core/common_runtime/executor.cc:1210] [/device:CPU:0] (DEBUG INFO) Executor start aborting (this does not indicate an error and you can ignore this message): INVALID_ARGUMENT: You must feed a value for placeholder tensor 'Placeholder/_0' with dtype int32
	 [[{{node Placeholder/_0}}]]
2/2 [==============================] - 1s 137ms/step - loss: 0.8602 - accuracy: 0.7400
[0.8602234125137329, 0.7400000095367432]